snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedConceptUpdateRequest.java

```diff
@@ -469,5 +470,5 @@ private void reactivateConcept(final TransactionContext context, final SnomedCon
 		}
 		return ids.build();
 	}
-
+	
```

Comment:
Is the permission missing here?

I can imagine two approaches:

* pull permissions up to superclasses, ie. set `Permissions.EDIT` on `SnomedComponentUpdateRequest`, also make the method final
* push down permissions to each leaf